src/ui/ui_style.c

```diff
@@ -32,18 +32,6 @@ int style_init(void) {
     lv_style_set_pad_right(&style_rootmenu, 0);
     lv_style_set_radius(&style_rootmenu, 0);
     lv_style_set_border_width(&style_rootmenu, 0);
-
```

Comment:
Why are these being removed?  Are they not referenced?

Reply:
Duplicate code... Just cleanup